src/time_series_cache.jl

```diff
@@ -116,6 +116,13 @@ _get_iterations_remaining(c::TimeSeriesCache) = c.common.iterations_remaining[]
 _decrement_iterations_remaining!(c::TimeSeriesCache) = c.common.iterations_remaining[] -= 1
 _get_resolution(cache::TimeSeriesCache) = get_resolution(_get_time_series(cache))
 
+struct TimeSeriesCacheKey
+    component_uuid::Base.UUID
+    time_series_type::Type{<:TimeSeriesData}
+    name::String
+    multiplier_id::Int
```

Comment:
Is there a more generic term for this?

Reply:
This name matched the related field in the time series.